gcp_variant_transforms/vcf_to_bq.py

```diff
@@ -464,20 +522,14 @@ def run(argv=None):
       table_suffix = ''
       if partitioner and partitioner.get_partition_name(i):
         table_suffix = '_' + partitioner.get_partition_name(i)
-      table_name = known_args.output_table + table_suffix
-      _ = (variants[i] | 'VariantToBigQuery' + table_suffix >>
-           variant_to_bigquery.VariantToBigQuery(
-               table_name,
-               header_fields,
-               variant_merger,
-               processed_variant_factory,
-               append=known_args.append,
-               update_schema_on_append=known_args.update_schema_on_append,
-               allow_incompatible_records=known_args.allow_incompatible_records,
-               omit_empty_sample_calls=known_args.omit_empty_sample_calls,
-               num_bigquery_write_shards=known_args.num_bigquery_write_shards,
-               null_numeric_value_replacement=(
-                   known_args.null_numeric_value_replacement)))
+      if (known_args.schema_version ==
+          variant_transform_options.BigQueryWriteOptions.V1):
+        _write_to_bigquery_v1(variants[i],
```

Comment:
seems rather weird that v1 and v2 writes are so far apart in code... how come?

Reply:
Yes, you are absolutely right. When the other tables are added, they will be surely added here. Done, moved the v1 schema here.